src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -0,0 +1,151 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 2;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 122px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 80px;
+    padding-top: 50px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTopLeft {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaTopRight {
+  grid-column: 6 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaBottomLeft {
+  grid-column: 2 / span 5;
+  grid-row: 2;
+}
+
+.gridAreaBottomRight {
+  grid-column: 8 / span 4;
+  grid-row: 2;
+}
+
+
+/* Component wrappers */
+
+.paragraph1Wrapper {
+  font: var(--font-body-large);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper {
+  font: var(--font-body-small);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper p:first-child {
+  margin-top: 0;
+}
+
+.paragraph2Wrapper p:last-child {
+  margin-bottom: 0;
+}
+
+.image1Wrapper {
+  position: relative;
+  top: 35px;
+}
+
+@media (--tablet-and-down) {
+  .paragraph1Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
+
+  .paragraph2Wrapper {
+    margin-right: 40px;
+    margin-top: 40px;
+  }
+
+  .image1Wrapper {
+    margin-left: 40px;
+    margin-top: 40px;
+    position: static;
+  }
+}
+
+
+/* Title block */
+
+.title {
+  font: var(--font-headline);
+  margin: 0;
+}
+
+
+/* Image block */
+
+.image1 {
+  height: auto;
```

Comment:
Is there a reason you needed to set this? This is what heights are set to by default.

Reply:
I came across a couple of articles that suggested setting `height: auto` to [maintain the aspect ratio](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)  of the image so that it would not be stretched or squashed and to [help browsers](https://html.com/attributes/img-height/)  render the page faster and more accurately but I can take it out since you're right on height being set to [auto by default](https://developer.mozilla.org/en-US/docs/Web/CSS/height).

Reply:
Thanks for the links to the articles, they were pretty interesting reads! Images are surprisingly one of the more complicated parts of HTML and CSS, since they have different default behavior from other kinds of elements and very often require us to preserve the aspect ratio in order to look nice, unlike most other elements in HTML.

I think we're safe to omit the `height: auto` here, since we're not explicitly setting the dimensions in the `<img>` tag itself. From your first article, it seems like we may actually want to start setting that, since it'll prevent the "pop in" problem, but I think this is something we can address later, especially after we've integrated with the CMS.

Reply:
Sounds great!